include/classes/Utility.class.php

```diff
@@ -376,10 +376,8 @@ public static function is_extension_in_list($filename, $extension_list)
 
 	public static function is_sha1($str)
 	{
-		if (strlen($str) == 40 && ctype_xdigit($str))
-			return true;
-		else 
-			return false;
+		return strlen($str) == 40 && ctype_xdigit($str);
 	}
+		
 }
 ?>
```

Comment:
Forgot to mention, add a EOF to the end of the file to get rid of the github warning. Thanks.

Reply:
Done